### ISSUE TITLE
bug: NVDA screen reader not reporting a file removal in File uploader

### ISSUE THREAD

**Issue by vv-account** [cds-snc/gcds-components#1363]
### Prerequisites | Prérequis

- [x] I have read the [Contributing Guidelines](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md). | J’ai lu les [Lignes directrices sur les contributions](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md).
- [x] I agree to follow the [Code of Conduct](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md). | J’accepte de me conformer au [Code de conduite](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md).
- [x] I have searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already report this problem, without success. | J’ai vérifié parmi [les tickets (issues) existants](https://github.com/cds-snc/gcds-components/issues) que ce sujet n’avait pas déjà été soumis.

### GC Design System Components Package and Version | Paquet et version des composants de Système de design GC

1.3.1

### Current Behavior | Comportement observé

When a file is removed from the File uploader, the NVDA screen doesn’t announce that the file has been removed. 

### Expected Behavior | Comportement attendu

When a file is removed from the File uploader, the NVDA screen should announce that the file has been removed. 

### System Info | Information sur le système

```shell
NVDA 2025.2
win 11 Ent – version 25H2
Edge Version 150.0.4078.105 (Official build) (64-bit)
npm 10.9.2
Vue 3.5.40
```

### Steps to Reproduce | Étapes pour reproduire le bogue

1) Navigate to the file-uploader/code page:

[https://design-system.canada.ca/en/components/file-uploader/](https://design-system.canada.ca/en/components/file-uploader/code/)

2) Scroll down to the Code builder section
3) Set the property multiple to true
4) Click on “Choose file” button and select multiple files > click on Open (to attach the files)
5) Turn on NVDA screen reader (I use version 2025.2)
6) Focus should be set on “Choose file” button
7) Tab to the first “Remove” button  -> “Remove button” announced
8) Tab to the second  “Remove” button  -> “Remove button” announced
9) Press “Enter” -> file removed but no NVDA announcement (the user doesn’t know whether the file has been removed or not)

Recording attached:

https://github.com/user-attachments/assets/5da268b1-a25e-462f-a390-375026b20b20

### Code Reproduction URL | URL de reproduction du code

https://design-system.canada.ca/en/components/file-uploader/code/

**Comment by adriannelee** [comment]
Hi @vv-account. Thanks for submitting this issue. We'll take a look at this next week!

**Comment by daine** [comment]
Hi @vv-account, a fix for this was just released in the newest version of GCDS v1.5.0 (thanks to @opensource-joe for their contribution!). We hope this solves the issue, but if you have any questions, concerns or further issues, feel free to open a new one and we'd be happy to take another look at it.

**Comment by vv-account** [comment]
Hi @daine,
Wonderful! Thank you very much for your update.
